Packs/Cisco-umbrella/Integrations/CiscoUmbrellaEnforcement/CiscoUmbrellaEnforcement.py

```diff
@@ -0,0 +1,188 @@
+import demistomock as demisto
+from CommonServerPython import *
+
+''' IMPORTS '''
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+
+class Client(BaseClient):
+    """
+    Client to use in the Cisco Umbrella Enforcement integration. Overrides BaseClient
+    """
+
+    def __init__(self, base_url, verify, proxy, api_key):
+        super().__init__(base_url=base_url, verify=verify, proxy=proxy, ok_codes=(200, 202, 204))
+        self.api_key = api_key
+
+    def get_domains_list(self, suffix: Optional[str] = '', request: Optional[str] = ''):
+        if request:
+            res = self._http_request('GET', full_url=request)
+        else:
+            res = self._http_request('GET', url_suffix=f"domains?customerKey={self.api_key}&{suffix}")
+        return res
+
+    def delete_domains(self, domain_name: str, domain_id: str):
+        res = ''
+        if domain_id:
+            res = self._http_request('DELETE', f"domains/{domain_id}?customerKey={self.api_key}",
+                                     return_empty_response=True)
+        if domain_name:
+            res = self._http_request('DELETE', f"domains?customerKey={self.api_key}&where[name]={domain_name}",
+                                     return_empty_response=True)
```

Comment:
what happens if the domain_id/domain_name does not exist? let's catch the exception and give a meaningful error.

Reply:
The exception from the api is pretty meaningful but I will add a try and except here.

Reply:
I meant that we want to catch and not return a war room entry of type error, but a regular entry stating the the domain name/id was not found.